readme.md

```diff
@@ -33,6 +33,12 @@ $ de --help
 
 It's `de` and not `del`, since `del` is already a builtin command on Windows.
 
+If preferred, a simple alias could fix this of course.
+
+```
+	alias del=de
```

Comment:
@schnittstabil I think the current is fine. People that don't use the common shells like bash/zsh usually knows the difference.